daemon/control.c

```diff
@@ -1060,7 +1038,11 @@ control_handle_message(control_t *control, const ControllerToDaemon *msg, int fd
 	} break;
 
 	case CONTROLLER_TO_DAEMON__COMMAND__PUSH_GUESTOS_CONFIG: {
-		control_handle_cmd_push_guestos_configs(msg, fd);
+		guestos_mgr_push_config(msg->guestos_config_file.data, msg->guestos_config_file.len,
```

Comment:
this may cause a null pointer dereference if guestos_config_file is not set in protobuf message, that is why we had the check in the first place. We agreed to set local variables in control_handle_cmd_push_guestos_configs() and give those to the guestos_mgr_push_config(). Why didn't you stood to this! Look in our internal issue SRMD-424, there is an initial code snippet for that.